src/components/Icon/index.js

```diff
@@ -34,13 +41,34 @@ class Icon extends PureComponent {
         const width = this.props.small ? variables.iconSizeSmall : this.props.width;
         const height = this.props.small ? variables.iconSizeSmall : this.props.height;
         const IconToRender = this.props.src;
-        return (
+
+        const rendered = (
             <IconToRender
                 width={width}
                 height={height}
                 fill={this.props.fill}
             />
         );
+
+        if (this.props.inline) {
+            return (
+                <View style={[{
+                    width, height,
```

Comment:
No inline styles, please.

Reply:
@parasharrajat how should I write those, once they come from props?

Reply:
https://github.com/Expensify/App/pull/6271#discussion_r757503485